keps/prod-readiness/sig-scheduling/4328.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 4247
+alpha:
+  approver: "TBD"
```

Comment:
Can we find a PRR approver?

Reply:
Will reach to `prod-readiness` channel after got a `lgtm` from sig.

Reply:
You can put me and ping me once you have SIG approval.